guides/v2.1/cloud/project/new-relic.md

```diff
@@ -50,66 +40,77 @@ We recommend adding at least one Admin user to your New Relic account to manage
 
 1.  Ask the new user to check their email for a New Relic notification with account information.
 
-
 See [Accounts](https://docs.newrelic.com/docs/accounts/accounts) in the New Relic documentation for information about user roles, managing user accounts, and changing account ownership.
 
+## Configure New Relic
 
+Pro accounts are preconfigured to use the New Relic service; however, Starter accounts must add the New Relic service to the `.magento.app.yaml` configuration file and assign the license key to an environment variable. You can apply the New Relic license key to a maximum of three active environments.
 
-## Add the New Relic extension to your project
+For Starter projects, check if the `.magento.app.yaml` file includes the New Relic extension in the `runtime` list. If not, then add the following:
 
-You must list the New Relic extension in the `.magento.app.yaml` file for your project. 
+> .magento.app.yaml
 
-```
+```yaml
 runtime:
-  extensions:
-    - newrelic
+    extensions:
+        - newrelic
 ```
 
-## Add New Relic APM to an environment
+## New Relic license key
 
-The {{site.data.var.ece}} plans support up to three instances of your New Relic APM license across all environments. We recommend adding a New Relic license to your Pro Staging and Production environments, and you can add the license to one other environment of your choice. 
+The {{site.data.var.ece}} Starter plans support up to three instances of your New Relic license across all environments. We recommend adding a New Relic license key to your Pro Staging and Production environments, and you can add the license to one other environment of your choice. 
 
 {:.bs-callout .bs-callout-info}
-If you have more than three active environments using the same New Relic license key, you need to remove a license variable from an existing environment.
-Contact your Magento technical account manager or New Relic sales (sales@newrelic.com) if you want to use New Relic on more than three environments.
+If you have more than three active environments using the same New Relic license key, you need to remove a license variable from an existing environment. Contact your Magento technical account manager or New Relic sales (sales@newrelic.com) if you want to use New Relic on more than three environments.
```

Comment:
I assume we need to remove sales from here - we can connect them once required with our TAM on NR side.